src/deploymentmanager.ts

```diff
@@ -523,17 +530,16 @@ export class DeploymentManager implements IDeploymentManager {
         data.push(`PCS_STORAGEADAPTER_DOCUMENTDB_CONNSTRING="${outputs.documentDBConnectionString.value}"`);
         data.push(`PCS_TELEMETRY_DOCUMENTDB_CONNSTRING="${outputs.documentDBConnectionString.value}"`);
         data.push(`PCS_TELEMETRYAGENT_DOCUMENTDB_CONNSTRING="${outputs.documentDBConnectionString.value}"`);
-        data.push(`PCS_IOTHUBREACT_HUB_ENDPOINT="Endpoint=${outputs.eventHubEndpoint.value}"`);
-        data.push(`PCS_IOTHUBREACT_HUB_PARTITIONS=${outputs.eventHubPartitions.value}`);
-        data.push(`PCS_IOTHUBREACT_HUB_NAME=${outputs.eventHubName.value}`);
-        data.push(`PCS_IOTHUBREACT_AZUREBLOB_ACCOUNT=${outputs.storageAccountName.value}`);
-        data.push(`PCS_IOTHUBREACT_AZUREBLOB_KEY="${outputs.storageAccountKey.value}"`);
-        data.push(`PCS_IOTHUBREACT_AZUREBLOB_ENDPOINT_SUFFIX=${storageEndpointSuffix}`);
         data.push(`PCS_ASA_DATA_AZUREBLOB_ACCOUNT=${outputs.storageAccountName.value}`);
         data.push(`PCS_ASA_DATA_AZUREBLOB_KEY="${outputs.storageAccountKey.value}"`);
         data.push(`PCS_ASA_DATA_AZUREBLOB_ENDPOINT_SUFFIX=${storageEndpointSuffix}`);
+        data.push(`PCS_AZUREBLOB_ACCOUNT=${outputs.storageAccountName.value}`);
```

Comment:
are these three variables (blob account, key, endpoint) used anywhere currently? If they aren't and we are just going to use them for telemetry, we should change them into 1 variable, the connection string. The format is AccountName={AccountName};AccountKey={AccountKey};EndpointSuffix={EndpointSuffix}

Reply:
I think they are used in ASA manager: https://github.com/Azure/remote-monitoring-services-dotnet/blob/master/asa-manager/WebService/appsettings.ini#L44.

Reply:
yes the asa_data ones are, but if we replace the iothubreact_ ones with a single connection string variable, we can transition asa to also use the connection string later, then delete the asa_data ones, which will save us 3 environment variables